docs/index.rst

```diff
@@ -0,0 +1,44 @@
+.. intake-dal documentation master file
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to Intake DAL (data access layer) plugin
```

Comment:
does this need adjustment to intake-hive with examples?

Reply:
fixed!